### ISSUE TITLE
Osmand Live updates POI only partially

### ISSUE THREAD

**Issue by Discostu36** [osmandapp/Osmand#8213]
POI is only updated on the map, not in the popup.

https://www.openstreetmap.org/node/292066349

Map: Germany Schleswig-Holstein Jan 2020
Osmand Live active

![Screenshot_20200110-082118_OsmAnd](https://user-images.githubusercontent.com/30999326/72149731-151b8680-33a4-11ea-87b7-8197861bef5b.jpg)

**Comment by vshcherb** [comment]
[Germany_schleswig-holstein_europe_20_01_00.obf.zip](https://github.com/osmandapp/Osmand/files/4092477/Germany_schleswig-holstein_europe_20_01_00.obf.zip)
This is a tricky bug. The data is correct in the obf, so "Restaurant is deleted" and no new POI is created because : **we don't support shop=security** (this could be another issue).

So context menu tries to find something in POI and it fails and it takes old POI (though it doesn't display it is deleted) . This is really an exceptional situation cause we should support all tags in POI that we support in rednering.

**Comment by Discostu36** [comment]
Another example: A POI changed from `amenity=restaurant` to `disused:amenity=restaurant`. All deleted tags are shown in pop up as if the restaurant was still open. At least search shows it as closed.

https://www.openstreetmap.org/node/5585390080

![Screenshot_20200807_162753](https://user-images.githubusercontent.com/30999326/89657312-2cd0d400-d8cd-11ea-89db-6b6efb08ab47.jpg)
![Screenshot_20200807_162826](https://user-images.githubusercontent.com/30999326/89657324-322e1e80-d8cd-11ea-846a-204c2cfc78de.jpg)

**Comment by vshcherb** [comment]
We don't support changing POI object itself, we support deleting POI. Though if POI is transformed from Bar -> Gym, you will be able to find Bar and Gym at the same time cause Bar will be loaded from the old map and Gym from new map.

This is quite rare situation and we don't how to support it yet.

**Comment by Discostu36** [comment]
I don't think that lifecycle prefixes are a rare situation. `disused:amenity` has 27,000 uses in the database.
